test/e2e/performanceprofile/functests/utils/pods/pods.go

```diff
@@ -207,7 +207,11 @@ func GetContainerIDByName(pod *corev1.Pod, containerName string) (string, error)
 	}
 	for _, containerStatus := range updatedPod.Status.ContainerStatuses {
 		if containerStatus.Name == containerName {
-			return strings.Trim(containerStatus.ContainerID, "cri-o://"), nil
+			containerID := containerStatus.ContainerID
+			// Split the container ID at the "://" delimiter, keeping the second part:
+			containerID = strings.SplitN(containerID, "://", 2)[1]
+			return containerID, nil
+
```

Comment:
this works, but it seems a textbook case of https://pkg.go.dev/strings#CutPrefix . Can we use `CutPrefix`?

Reply:
good point
/lgtm cancel

Reply:
If the prefix is not found that also means it's junk , as we are  checking object of v1.ContainerStatus where it's of   '<type>://<container_id>'.

Reply:
fair enough! I still think `CutPrefix` can help; but even if it is not, and we need to use `Split` or `SplitN`, we need a bit more robustness check

Reply:
updating the patch to use CutPrefix